apollo-portal/src/main/resources/static/vendor/iconfont/iconfont.css

```diff
@@ -0,0 +1,31 @@
+@font-face {
+  font-family: "iconfont"; /* Project id 2742392 */
+  src: url('http://at.alicdn.com/t/font_2742516_pik2lmzsw08.woff2?t=1628826396308') format('woff2'),
```

Comment:
should we reference the iconfont.woff2, iconfont.woff, iconfont.ttf in the projects instead of external resources?